spec/tapioca/dsl/compilers/active_record_columns_spec.rb

```diff
@@ -941,236 +941,6 @@ def cost=(value); end
                 assert_includes(rbi_for(:Post), expected)
               end
 
-              it "discovers custom type from signature on cast method" do
```

Comment:
I removed the case for using a custom type to leave only one case, since the detailed pattern is covered by the tests for the helper.